cypress/integration/mega-menu-dropdown.spec.js

```diff
@@ -14,4 +20,16 @@ describe('Mega menu dropdown', () => {
     cy.get(megaMenuDropdownSelector).should('be.visible')
     cy.get(megaMenuDropdownContentSelector).should('be.visible')
   })
+
+  it('should contain correct links', () => {
+    // find all links in megamenu
+    cy.get('q-content-menu a').each($a => {
+      // skip the titles
+      if ($a.attr('class').indexOf('link_title') == -1) {
+        const url = $a.attr('href').replace(originRegex, '')
+        // url should match what is in the sitemap
+        expect(url).to.be.oneOf(sitemapUrls)
+      }
+    })
```

Comment:
Just a couple of small possible improvements:

1. Changed `$a` to descriptive variable name, like `link`
2. Added parens in arrow function (https://eslint.org/docs/rules/arrow-parens)
3. Replaced `indexOf` with `includes` (https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v28.0.2/docs/rules/prefer-includes.md)

```suggestion
    // find all links in megamenu
    cy.get('q-content-menu a').each((link) => {
      // skip the titles
      if (!link.attr('class').includes('link_title')) {
        const url = link.attr('href').replace(originRegex, '')
        // url should match what is in the sitemap
        expect(url).to.be.oneOf(sitemapUrls)
      }
    })
```

Reply:
thank you